dash/frontend/src/app/modules/private/pages/cluster/kubesec/kubesec.component.ts

```diff
@@ -125,7 +125,7 @@ export class KubesecComponent implements OnInit, OnDestroy {
           }
         }
       } else {
-        this.alertService.danger('Could not get pods in ' + this.selectedNamespaces);
+        this.snackBar.open('Could not get pods in ', 'Close', { duration: 2000});
```

Comment:
This now just says 'Could not get pods in' Rather than 'Could not get pods in [namespace]'